keps/sig-network/3070-reserved-service-ip-range/README.md

```diff
@@ -421,33 +433,46 @@ The allocator using this feature will have a `dynamic_range_offset` value differ
 
 ###### Does this feature depend on any specific services running in the cluster?
 
+No
 ### Scalability
 
 ###### Will enabling / using this feature result in any new API calls?
 
-
+No
 ###### Will enabling / using this feature result in introducing new API types?
 
+No
 ###### Will enabling / using this feature result in any new calls to the cloud provider?
 
+No
+
 ###### Will enabling / using this feature result in increasing size or count of the existing API objects?
 
+No
 ###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
 
+No
```

Comment:
Theoretically, it would be possible that IP allocations take slightly longer, of in the event of some sort of bug, a lot longer. I see metrics above around error rates, do we track IP allocation time today (I suspect NOT).

Reply:
There are no metrics about IP allocation time, I've considered to add those metrics but the problem is that they are tied to the implementation of the allocator Interface, in this case the bitmap, but the same logic is used for nodeports, so you can't differentiate IPs from NodePorts when you are using the bitmaps. However, sounds a good metrics to add in the new API for Service allocation.

It can take longer, but this only changes the strategy to find a free number into a bitmap, with current change instead of doing a linear search in the whole bitmap with size N, we split the bitmap by a constant. We do the linear search on the N-K, and if not found , do the search in the remaining K bits, and K is always a small number

Reply:
Agree with Antonio.